chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -853,8 +857,11 @@ mod test {
             let now = 1590490084;
             let t = kp.verify(&*ENCLAVE_CERT_VERIFIER, now).unwrap();
 
+            let mut public_key = [0u8; 65];
+            public_key.copy_from_slice(certificate);
```

Comment:
certificate here is the full X509 payload, not just the pubkey, so this is incorrect

Reply:
I didn't change any logic here. Only changed this from `Vec<u8>` to `[u8; 65]`. @yihuang Any comments?